services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -290,19 +296,21 @@ public void waypointTargetsList() {
 
   @Test
   public void annotationsString() {
-    String annotationsStr = ANNOTATION_MAXSPEED + ";" + ANNOTATION_DURATION + ";" + ";";
+    String annotationsStr = ANNOTATION_MAXSPEED + "," + ANNOTATION_DURATION + "," + ",";
```

Comment:
This is not a valid use case for `annotations`, we should remove the commas `","` from the `annotationsStr`. `annotations` doesn't accept trailing empty values.

Reply:
fixed